mappings/net/minecraft/screen/ScreenHandler.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/class_71 net/minecraft/screen/ScreenHandler
+	FIELD field_2731 revision S
```

Comment:
What is the "revision" field about? This word on its own is confusing.

Reply:
Honestly not sure

Reply:
Doesn't really matter since it's the same thing as the one in yarn - https://github.com/FabricMC/yarn/blob/bd3e21e1cd577dc1b0f4ec41cc88d8933e78c7e1/mappings/net/minecraft/screen/ScreenHandler.mapping#L137